src/voicefocus/VoiceFocusModelComplexity.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/* istanbul ignore next */
+import { ModelVariant as VoiceFocusModelComplexity } from '../../libs/voicefocus/types';
```

Comment:
Sorry I commented at the wrong place above in a demo code [here](https://github.com/aws/amazon-chime-sdk-js/pull/1476/files#r680209054), but, I meant it for this import-export.

Reply:
@devalevenkatesh no, it doesn't really need any documentation — it's a bounded set of strings, and it's already explained in the main VF guide.

Reply:
Cool, thanks for the confirmation.